lobby_server.yaml

```
@@ -18,7 +18,7 @@
     Join the TripleA ranked ladder by posting here:
     https://forums.triplea-game.org/topic/1068/invitation-requests-to-the-new-ladder-and-tournament-site
   error_message:
-- version: 1.10.0.0
+- version: 1.10
```

The JSON schema parser needs a bit of help to distinguish a `string` from a `number` in this case:

```suggestion
- version: "1.10"
```